cfme/ansible/repositories.py

```diff
@@ -110,6 +117,12 @@ def db_object(self):
     def playbooks(self):
         return self.collections.playbooks
 
+    def get_detail(self, title, field, refresh=False):
```

Comment:
**Required/Question**

@psav @mfalesni @jkrocil Would like your opinion here.
It was my understanding part of the effort to convert to WT was removing `load_detail`/`get_detail` methods such as this, since with WT this can instead be achieved with a single line `view.entities.repository_options.get_text_of(field)`.

Looks like this is only used in a wait_for, where the refresh click can be passed as fail_func.

I'd prefer to not have `get_detail` methods sporadically through the API.

Reply:
@mshriver Agreed.

This issues exists in a few other places of our codebase already, though:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/infrastructure/host.py#L202
https://github.com/ManageIQ/integration_tests/blob/master/cfme/containers/provider/__init__.py#L572
https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/vm.py#L277
...

They should be fixed all at once, but in a separate PR. I'm in favor of a bit of tech-debt for the sake of more stable RC.